test/unit/get-config-test.js

```diff
@@ -613,116 +612,52 @@ describe('determineRuleConfig', function () {
   });
 });
 
-describe('getConfigForFile', function () {
-  it('Merges the overrides rules with existing rules config', function () {
-    let config = {
-      rules: {
-        foo: 'bar',
-        baz: 'derp',
-      },
-      overrides: [
-        {
-          files: ['**/templates/**/*.hbs'],
-          rules: {
-            baz: 'bang',
-          },
-        },
-      ],
-    };
-
-    let expectedRule = {
-      foo: 'bar',
-      baz: 'bang',
-    };
-    let actual = getConfigForFile(config, { filePath: 'app/templates/foo.hbs' });
-
-    expect(actual.rules).toEqual(expectedRule);
-  });
-
-  it('Returns the correct rules config if overrides is empty/not present', function () {
-    let config = {
-      rules: {
-        foo: 'bar',
-        baz: 'derp',
-      },
-      overrides: [],
-    };
-
-    // clone to ensure we are not mutating
-    let expected = JSON.parse(JSON.stringify(config));
-
-    let actual = getConfigForFile(config, 'app/templates/foo.hbs');
-
-    expect(actual.rules).toEqual(expected.rules);
-
-    delete config.overrides;
-
-    actual = getConfigForFile(config, 'app/templates/foo.hbs');
+describe('resolveProjectConfig', function () {
+  it('should return an empty object when options.config is set explicitly false', function () {
+    const config = resolveProjectConfig({ config: false });
 
-    expect(actual.rules).toEqual(expected.rules);
+    expect(config).toEqual({});
   });
+});
 
-  it('Merges the overrides rules from multiple overrides with existing rules config', function () {
+describe('getProjectConfig', function () {
```

Comment:
This file is a bit noisier in the diff than expected. There were some tests that didn't use `getConfig` underneath the `getConfig` describe block. I moved them to their own describe blocks but they should otherwise be unchanged.